### PR TITLE
SG-35935 first phase python2 removing

### PR DESCRIPTION
**Pull request by eduardoChaucaGallegos**
- Six ensure functions replaced with function from sgutils

### CODE REVIEW

**Review by julien-lang — CHANGES_REQUESTED** [review]
I don't think we should "patch" the third-party libraries we bundle in Vendors/.
Can we update the library instead maybe?

Or was these libraries were manually patches before for Python 3 support?

**Comment by carlos-villavicencio-adsk** [comment]
> I don't think we should "patch" the third-party libraries we bundle in Vendors/. Can we update the library instead maybe?
> 
> Or was these libraries were manually patches before for Python 3 support?

Julien is right. This component has its own six [here](https://github.com/shotgunsoftware/tk-framework-desktopclient/tree/master/Vendors). Maybe we should leave it as it is for now. It's also required to separate the libraries per python version at some point.

File: Vendors/bin/wsdump.py
Comment (julien-lang) [inline Vendors/bin/wsdump.py:1]: I think this file is part of the websocket_client third party (only call from there) so I think we should drop changes from this file too.

**Review by julien-lang — APPROVED** [review]

**Review by carlos-villavicencio-adsk — APPROVED** [review]
LGTM

**Pull request merged**